tests/__init__.py

```diff
@@ -1,38 +0,0 @@
-import numpy as np
-
-attrs_to_check = """
-scheme
-scheme_version
-sample
-subtype
-are_subtypes_consistent
-n_tiles_matching_all
-n_tiles_matching_all_expected
-n_tiles_matching_positive
-n_tiles_matching_positive_expected
-n_tiles_matching_subtype
-n_tiles_matching_subtype_expected
-qc_status
-""".strip().split('\n')
-
-exp_fasta_cols = ['tilename', 'contig_id', 'refposition', 'seq', 'subtype', 'match_index', 'is_revcomp',
-                  'is_pos_tile', 'sample', 'file_path', 'scheme', 'scheme_version', 'qc_status', 'qc_message']
-
-exp_fastq_cols = ['tilename', 'refposition', 'subtype', 'seq', 'freq', 'is_pos_tile', 'is_kmer_freq_okay',
-                  'sample', 'file_path', 'scheme', 'scheme_version', 'qc_status', 'qc_message']
-
-
-def check_subtype_attrs(*sts):
-    for a in attrs_to_check:
```

Comment:
just a small thing, but perhaps a more descriptive variable name for a?

Reply:
Changed in 9511966bb5c1825cc60001ba162d0f5045a8751a